appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -16,18 +17,20 @@ else{
   throw "Unrecognized repository provider: $repoProvider. Set the APPVEYOR_REPO_PROVIDER environment variable to a supported provider. Supported providers are: gitHub."
 }
 
-function Rebase-Uri($originalURI){
-  $replace = $originalURI.IndexOf("file:///")
-  if($replace -ne  -1){
-      $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString("file:///".Length))
-      $replaceLength = $caseSensitiveUri.IndexOf($projectSlug)
-      if($replaceLength -ne -1){
-          $builder.Path = ($repoName, $repoCommit, $caseSensitiveUri.SubString($replaceLength+$projectSlug.Length+1) -join '/')
-          return $builder.ToString()
-      }
+function Rebase-Uri($originalUri){
+  if($originalUri){
+    $fileUriPrefix = "file:///"
+    if($originalUri.StartsWith($fileUriPrefix)){
+        $caseSensitiveUri = Get-CaseSensitivePath $originalUri.SubString($fileUriPrefix.Length)
+        $projectSlugIndex = $caseSensitiveUri.IndexOf($projectSlug)
+        if($projectSlugIndex -ne -1){
+            $builder.Path = ($repoName, $repoCommit, $caseSensitiveUri.SubString($projectSlugIndex+$projectSlug.Length+1) -join '/')
```

Comment:
Ok, that tells me something I didn't know: the priority of the comma operator is higher than that of `-join`.

In that case, the outer set of parentheses should not be necessary:

```
$builder.Path = $repoName, $repoCommit, $caseSensitiveUri.SubString($projectSlugIndex+$projectSlug.Length+1) -join '/'
```